### PR TITLE
Feature/protected routes

### PR DESCRIPTION
**Pull request by VideyKohaku**
added protected landlord route from guest user
finished tenant landing page

### CODE REVIEW

File: src/redux/features/header/headerSlice.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { createSlice } from "@reduxjs/toolkit";
+import { createSlice, current } from "@reduxjs/toolkit";
 
 const initialState = {
```
Comment (datdev2409) [inline src/redux/features/header/headerSlice.js]: why do we need to store the state of both buttons?

Comment (datdev2409) [reply src/redux/features/header/headerSlice.js]: Suggest: use 1 variable to store the current view mode is landlord or tenant. And using that variable to active the corresponding button. Example:
 `
const initialState = {
         viewMode: "landlord"
}
`

File: next.config.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    images: {
+        domains: ['datnguyen2409-bkrental-dev.s3.ap-southeast-1.amazonaws.com'],
+    },
+}
```
Comment (datdev2409) [inline next.config.js]: add EOF here

File: src/redux/features/header/headerSlice.js (hunk 1 above)
Comment (VideyKohaku) [reply src/redux/features/header/headerSlice.js]: Changed the headerSlice into Config slice and created a new state to keep track the button

**Review by datdev2409 — APPROVED** [review]

**Pull request merged**